Automattic/wp-calypso
# Add author to post item

This PR adds the author to the post item card if the site is a multi-user site, or All Sites mode is selected with posts from multiple users.

All Sites mode, multiple authors:
![screen shot 2017-08-25 at 10 56 26 pm](https://user-images.githubusercontent.com/1017839/29732385-a88b6a38-89e8-11e7-8ae3-ea7bf730eb1e.png)

Single site mode, multiple authors:

![screen shot 2017-08-25 at 11 03 09 pm](https://user-images.githubusercontent.com/1017839/29732616-a13f2818-89e9-11e7-9a34-47127f798bf1.png)

Single site mode, single author:

![screen shot 2017-08-25 at 11 04 23 pm](https://user-images.githubusercontent.com/1017839/29732654-c4cb1a6c-89e9-11e7-8ab9-9da339eecb1f.png)


This PR is part of a larger epic and behind the posts/post-type-list feature flag.
See p8F9tW-hL-p2.

To test:

- Check out this branch
- ENABLE_FEATURES=posts/post-type-list npm start
- Navigate to Blog Posts
- Select All Sites
- Verify that the author is visible if posts are from multiple authors
- Verify that the Site icon and site name are visible
- Select a specific site
- Verify that the author is visible if the site has multiple authors
- Verify that the Site icon and site name are not visible
